tests/input/cpp/indent_paren_after_func_def.cpp

```diff
@@ -0,0 +1,5 @@
+void SomeFunction
+(
+int const aTest,
+int const aResult
+);
```

Comment:
It actually works the same for definition and declaration (due to how the code is already structured), but I understand what you're saying.